### ISSUE TITLE
B027 false positive with abc.abstractclassmethod

### ISSUE THREAD

**Issue by Skylion007** [charliermarsh/ruff#4285]
Adding an abstractmethod decorator does not make sense if an abstractclassmethod decorator is used instead:
When doing an automated fixes to the PyTorch codebase I ran into an issue in the wild:

```
class Foo(metaclass=abc.ABCMeta):
    @abc.abstractclassmethod
    def read(self): 
        pass
```
should not be flagged for instance (but currently is).  This should be an easy exclusion to add to the rule and a good first issue.

**Comment by charliermarsh** [comment]
Just gonna fix this real quick.
